### ISSUE TITLE
NN "torso"?

### ISSUE THREAD

**Issue by drozzy** [deepmind/acme#25]
What is a "torso"?
I encountered it in the paper, and also here:
https://github.com/deepmind/acme/blob/f8867fe383c591d00f873323b308cd4e70f9f818/acme/tf/networks/continuous.py#L38

Are there are parts that you use as well? Just curious about terminology.
Thanks.

**Comment by kjacek** [comment]
"When stacking modules it is often, though not always, helpful to distinguish between what are often called torso, head, and multiplexer networks. Note that this categorization is purely pedagogical but has nevertheless proven useful when discussing network architectures.

Torsos are usually the first to transform inputs (observations, actions, or a combination) and produce what is commonly known in the deep learning literature as an embedding vector. These modules can be stacked so that an embedding is transformed multiple times before it is fed into a head."
https://github.com/deepmind/acme/blob/master/docs/components.md

**Comment by aslanides** [comment]
Yes, this is a common nomenclature we use to distinguish between layers/modules that are low in the 'stack', e.g. a convnet for vision processing or some other embedding network (torsos) and layers that are higher in the stack, which normally produce outputs (e.g. a value 'head'). We separate these to keep things composable, so e.g. you could have an IMPALA agent using a (logits, value) head on top of a simple MLP torso or a full ResNet torso. Hope that helps!
